sdk_api.json

```diff
@@ -43468,13 +43468,27 @@
         "declKind": "Import",
         "moduleName": "Sentry"
       },
+      {
+        "kind": "Import",
+        "name": "Darwin",
+        "printedName": "Darwin",
+        "declKind": "Import",
+        "moduleName": "Sentry"
+      },
```

Comment:
`m`: I think we should be able to undo all the changes in the api.json files, because the current solution doesn't need any API changes.

Reply:
Ah that makes sense thanks.